examples/kops-with-vpc/README.md

```diff
@@ -13,18 +15,21 @@ We then use `kops` to create the kubernetes cluster in this existing VPC, where
 3. EBS volumes
 4. IAM instance profiles, role policies
 5. Keypairs for every kubernetes component
-6. Files and secrets required to configu kubernetes clusters
+6. Files and secrets required to configure kubernetes clusters
 7. Security groups
+
 ...and modifies the tags on existing subnets to make things easier for discovery within the kubernetes cluster
 
 ## Environment creation and deployment
-`NOTE`: For usage with `aws-env` have each command below execute in the form `aws-env -p <your-profile> -- <the-command>`.
+`NOTES`:
+  - For usage with `aws-env` have each command below execute in the form `aws-env -p <your-profile> -- <the-command>`.
+  - The word `template` in the snippets below refers to your target environment. As an example, `template/env` is provided under the respective directories for reference. Feel free to make directories named after your target environment like `production`, `staging`, etc. in both `terraform` and `kubernetes` sub-directories to support multiple target environments.
```

Comment:
I think this can be worded a more strongly: users _should_ copy the template to their own environment name and then modify it, rather than use it as it is.